### PR TITLE
My firstfile added

### PR DESCRIPTION
**Pull request by JyotiGujjari**
Automation Code changes

### CODE REVIEW

File: Jyoti/firstfile.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hi
```
Comment (sqatools) [inline Jyoti/firstfile.txt:1]: Can you please improve the code?

**Comment by R14041989** [comment]
hello

**Comment by JyotiGujjari** [comment]
test

**Review by sqatools — APPROVED** [review]
Lookig good to me

**Pull request merged**